pegasus/sites.v3/code.org/public/privacy/index.md

```diff
@@ -3,13 +3,13 @@ title: Privacy Policy
 theme: responsive
 ---
 
-[<img src="/shared/images/student_privacy_pledge.png" width="120" target="_blank">](http://studentprivacypledge.org/)
-
 # Privacy Policy
 Date of Last Revision: April 4, 2024
 
 You can review the previous Privacy Policy [here](/privacy-jun2023).
 
+[<img src="/shared/images/student_privacy_pledge.png" width="120" target="_blank" style="margin-top: 1rem">](http://studentprivacypledge.org/)
```

Comment:
I also moved the Student Privacy Pledge image further down on the page to make the page flow better and fix a spacing issue:
| Before | After |
| ---- | ---- |
| ![Before](https://github.com/code-dot-org/code-dot-org/assets/9256643/9293047a-6745-46cc-8dac-619d1d833047) | ![After](https://github.com/code-dot-org/code-dot-org/assets/9256643/d110a033-ba41-4db8-9c36-b5eb22d19ce1) |